content/en/docs/collector/builder.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Builder
+weight: 25
+---
+
+The OpenTelemetry Collector Builder (ocb) is a command-line interface tool that
+generates sources and binaries for OpenTelemetry Collector
+[distributions](../distributions).
+
+This tool is currently being used to build both the core and the contrib
+distributions, and can be used to build custom distributions as well, allowing
+users to cherry-pick which components they need for their production usage,
+potentially even including their own custom components.
+
+## Installing
+
+You can download `ocb` from the OpenTelemetry Collector's
+[release page](https://github.com/open-telemetry/opentelemetry-collector/releases)
+on GitHub. A container image is **NOT** available at this time. Prefer to use
+the same version number that you'll use in your manifest, as the builder is only
+guaranteed to work with the OpenTelemetry Collector API matching the builder's
+version.
+
+## Manifest
+
+The central part of a custom distribution is the distribution's manifest. This
+manifest defines the metadata for the distribution and lists all components that
+should be included.
+
+This is an example `manifest.yaml`, containing only a couple of components:
+
+```yaml
+dist:
+  module: github.com/open-telemetry/opentelemetry-collector-releases/sidecar
+  name: sidecar
+  description: OpenTelemetry Collector Sidecar version
+  version: 0.73.0
+  output_path: ./_build
+  otelcol_version: 0.73.0
+
+receivers:
+  - gomod: go.opentelemetry.io/collector/receiver/otlpreceiver v0.73.0
+
+exporters:
+  - gomod: go.opentelemetry.io/collector/exporter/otlpexporter v0.73.0
+  - gomod: go.opentelemetry.io/collector/exporter/otlphttpexporter v0.73.0
+
+processors:
+  - gomod: go.opentelemetry.io/collector/processor/batchprocessor v0.73.0
+```
+
+> Tip: you can keep the manifest in a source code repository and generate the
+> binaries via your CI/CD pipeline
+
+## Building the custom distribution
```

Comment:
While slightly out of scope for building a collector, I'd like to see a minimal dockerfile for packaging the output a build.
This was the only other thing I looked for in the releases repo when I needed to build my own instance.
While simple enough to write on your own, it's nice to have a confirmation of the (lack of) required dependencies.

Reply:
semi-related, cross platform builds?
also static / CGO_ENABLED=0 ?

Reply:
Good idea, would you mind adding that as a follow-up improvement to this, once it's merged? Given the number of approvals on this already, I would prefer to have this improvement on its own PR.